scripts/release/preparation_before_release.sh

```diff
@@ -22,8 +22,8 @@ set -e
 
 LOCAL_SVN_DIR=local_svn_dir
 ROOT_SVN_URL=https://dist.apache.org/repos/dist/
-DEV_REPO=dev/incubator
-RELEASE_REPO=release/incubator
+DEV_REPO=dev
```

Comment:
we sure about this location? (anyway the next RM has his/her job cut out :))

Reply:
yes, TLP just locate in dev/release without incubator